src/display/annotation_layer.js

```diff
@@ -2608,18 +2615,17 @@ class AnnotationLayer {
       const element = AnnotationElementFactory.create({
```

Comment:
All the properties except the first one are constant, so maybe we could just create a param Object before the loop and then just change its data property before calling `create`.

Reply:
Right, that should work fine.